README.md

```diff
@@ -2,103 +2,103 @@
+and
+[`colcon`](https://catkin-tools.readthedocs.io/en/latest/installing.html)
+should all work.
+
+### Autoware
+
+The `sd_autoware` package included in this repository provides an
+interface to be able to control the StreetDrone model using the
+[Autoware.AI](https://catkin-tools.readthedocs.io/en/latest/installing.html)
+open-source self-driving car stack.
+
+By default this package is not built to prevent any problems when
+Autoware.AI is not available. If it is available and you do want to
+use the package, remove the file
+`streetdrone_model/sd_autoware/CATKIN_IGNORE` and build again.
+
+## Launch the simulation
+The following launches the vehicle model in Gazebo and RViz for
+visualizing the sensors' output:
+
 ```
-roslaunch sd_robot sd_twizy_empty.launch
+roslaunch sd_bringup sd_twizy_gazebo.launch
 ```
 
-<p align="center"> 
-<img src="streetdrone_model/sd_docs/imgs/sd.png">
+<p align="center"> <img src="streetdrone_model/sd_docs/imgs/sd.png">
 </p>
 
-You might need to update your ignition-math version `sudo apt upgrade libignition-math2`
+There are several more launch files included to provide a more
+interesting surrounding for the vehicle; try for instance:
 
-##### Display the robot only in Gazebo:
-To only launch the model in Gazebo, do:
 ```
-roslaunch sd_robot sd_twizy_empty_gazebo.launch 
+roslaunch sd_bringup sd_twizy_gazebo_default.launch
 ```
 
-##### Known Issues:
-* On Gazebo7 the model logs an error on the p-gain of the wheels' velocity controllers. This does not affect the performance of the model and has only been detected when running the gazebo-ros-control package for Gazebo7. If you choose to build and run the model in Gazebo8 with gazebo8-ros-control, the problem disappears.
-
-### Controlling the Robot:
-#### Joystick:
-The robot supports the generic Linux [joystick](http://wiki.ros.org/joy) controllers.  
-To publish the `cmd_vel` topic [[geometry_msgs/Twist]](http://docs.ros.org/api/geometry_msgs/html/msg/Twist.html), clone or copy the contents of the ROS package [teleop_twist_joy](https://github.com/ros-teleop/teleop_twist_joy) at the `~/catkin_ws/src` folder of your catkin workspace. Then build the package by doing `catkin build` or `catkin_make`, depending on what you previously used to build the simulation model. After you have built the package, do: `source devel/setup.bash`  
-To launch the controller, do `roslaunch teleop_twist_joy teleop.launch`
+or:
 
-##### A. Microsoft Xbox 360 Controller for Linux:
-To control the vehicle, keep the Xbox Guide button pressed (Index 8) and use the Left Stick (Index 9) to navigate
+```
+roslaunch sd_bringup sd_twizy_large_city.launch
+```
 
-##### B. Logitech Wireless Gamepad F710 (DirectInput Mode):
-To control the vehicle, keep the Back button (Index 8) pressed and use the Start button (Index 9) to navigate
+### Display the robot only in Gazebo
+To only launch the model in an already running instance of Gazebo, do:
+```
+roslaunch sd_bringup sd_twizy_spawn.launch
+```
```

Comment:
E.g.: Open an empty world model with `roslaunch gazebo_ros empty_world.launch` and then launch the model with  `roslaunch sd_bringup sd_twizy_spawn.launch`